decentraland/unity-renderer
# Feat/passport about tab

## What does this PR change?

Fix #3315
Adds the about tab in the new passport

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/passport-about-tab&ENABLE_new_avatar_flow=
2. Open the passport of a regular (non-guest) user
3. Verify that the buttons in the header change the section correctly
4. Verify that the description is correctly displayed if the user has a description
5. Verify that no description panel appears if there isn't a description set
6. Verify that the equipped wearables are correctly displayed
7. Verify that hovering and clicking on a wearable opens the marketplace page

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
